10-sharing-data.Rmd

```diff
@@ -5,6 +5,172 @@
+
+There's so many excellent reasons to put your data in a repository whether or not a journal requires it:  
+
+**Sharing your data...**  
+
+1. Makes your project more transparent and thus more likely to be trusted and cited. In fact one study found that articles with links to the data used (in a repository) were cited more than articles without such information or other forms of data sharing [@colavizza_citation_2020].
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Another researcher is downloading the data from a repository and says ‘These insights are so exciting! I can’t wait to look into this data even more!’", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g33bf0789107_201_144")
+```
+
+2. Helps your relieve your own workload so your email inbox isn't loaded by requests you probably don't have time to respond to.
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Ruby is reading a journal article with data and code she is interested in. The journal article says ‘Code and data are available upon request by email’. Ruby sends an email that says ‘ The email is going to an inbox with 999,999,565473 emails in it and it is labeled ‘the corresponding author’s inbox’.", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g33bf0789107_201_335")
+```
+
+3. Allows others to gain even more insights from your data which shows funders that your data will be used to its maximum potential.
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Ruby has uploaded her data to a repository and now its being used by many other researchers. Ruby says to her funders, represented as a bank, ‘The data you funded is getting so much mileage!’", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g33bf0789107_201_529")
+```
+
+4. It also provides more opportunities for others to replicate your results, which could help advance not only your career, but our understanding of science and medicine.
+
+## Benefits of data sharing
+
+In addition to these benefits to yourself, data sharing has other far reaching benefits. It can help support faster advances in science and medicine, by reducing the need to collect new data, which reduces costs, time and effort, including the effort and burden that is required to collect data on or from patients.
+
+It also helps support researchers at institutes that do not have as many resources to collect data.
+
+Ultimately it can also therefore help patients benefit from research faster, as faster advances can be made through more efficient research.
```

Comment:
```suggestion
Ultimately it can therefore help patients benefit from research faster, as faster advances can be made through more efficient research.
```

Reply:
used also in the sentence before